acir/src/circuit/opcodes.rs

```diff
@@ -7,7 +7,7 @@ use crate::BlackBoxFunc;
 use serde::{Deserialize, Serialize};
 
 #[derive(Clone, PartialEq, Eq, Serialize, Deserialize, Hash, Copy, Default)]
-pub struct BlockId(u32);
+pub struct BlockId(pub u32);
```

Comment:
Would this be implemented with a `from<u32>` so the way it is stored isn't leaked to consumers/creators?

Reply:
I don't see the problem to leak this.

Reply:
No it won't change, I would not put anything else inside an id.